homeassistant/components/starlink/coordinator.py

```diff
@@ -53,7 +56,10 @@ async def _async_update_data(self) -> StarlinkData:
                 status = await self.hass.async_add_executor_job(
                     status_data, self.channel_context
                 )
-                return StarlinkData(*status)
+                location = await self.hass.async_add_executor_job(
+                    location_data, self.channel_context
+                )
+                return StarlinkData(location, *status)
```

Comment:
```suggestion
                return StarlinkData(*location, *status)
```
Don't we need to unpack as it does with status?

Reply:
It works fine on my end without unpacking it. It would throw an exception otherwise right?